modules/ROOT/pages/box/box-connector-examples.adoc

```diff
@@ -250,6 +250,260 @@ output application/json
 </mule>
 ----
 
+== Anypoint Studio Example: Upload File
+
+Since the Mule 4 version of Box connector doesn't provide upload file operation, this example demonstrates how to upload a file to Box.
+
+. Create a new Mule Project in Anypoint Studio
++
+. Create a folder called 'input' under src/main/resources directory
++
+. Copy the file you want to upload to the 'input' directory
++
+. Configure the HTTP Listener by adding a new HTTP Global Element. Click on the Global Elements tab at the base of the canvas. In the Global Configuration Elements screen, click Create. In the Choose Global Type wizard, expand Connector Configuration and select HTTP Listener and click Ok. In the HTTP Listener screen configure the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter|Value
+|Name|HTTP_Listener_config
+|Protocol| HTTP
+|Host| localhost
+|Port| 8081
+|===
++
+. Drag an HTTP Listener onto the canvas and configure the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name |HTTP Listener
+|Connector configuration | Select the HTTP Listener Configuration that had been created previously
+|Path |`/boxUploadFile`
+|===
++
+. Login to Box and Generate the Box developer token which is active for 60 minutes
+. Now set this Developer Token using a DataWeave component. Drag the DataWeave component next to the HTTP Listener and use the script below.
++
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/java
+		//Set the generated developer token here 
+var developerToken = "{developer Token}"
+---
+"Bearer " ++ developerToken
+----
++
+. Drag Set Variable onto Canvas and configure the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name| Save developer token
+|Name |developerToken
+|Value | #[payload]
+|===
++
+. Configure the File connector by adding a new File Global Element. Click on the Global Elements tab at the base of the canvas. In the Global Configuration Elements screen, click Create. In the Choose Global Type wizard, expand Connector Configuration and select File Config and click Ok. In the File Config screen configure the following parameter:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Working Dircetory | ${mule.home}/apps/${app.name}/
+|===
++
+. Drag a Read File operation onto canvas and configure the following parameters:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name |Read File
+|Connector configuration | Select the File Configuration that had been created previously in Global Elements
+|File Path |`input/yourFileName`
+|===
++
+. Now you need to transform this payload to multipart/form-data type of request. You can also specify a folder to which will be file uploaded vby setting the ID of parent folder. Drag a DataWeave component next to the Read File component and use the script below.
```

Comment:
Delete 'Now you need to' as it is unecessary. Rewrite the following sentence: Specify a folder to be uploaded by setting the ID of the parent folder.

Reply:
done